fluent-package/msi/assets/fluent-package-prompt.bat

```diff
@@ -1,3 +1,4 @@
 @echo off
 set PATH="%~dp0\bin";%PATH%
+set PATH="%~dp0";%PATH%

```

Comment:
`%~dp0\bin` is still needed too.

Reply:
For `fluent-cat`, `fluent-ctl` and so on.

Reply:
> %~dp0\bin is still needed too.

`%~dp0\bin` is not removed in this PR. Am i missing?

Reply:
Sorry, I overlooked it.